R/test-files.R

```diff
@@ -27,17 +27,20 @@ test_env <- function() {
 #' @return the results as a "testthat_results" (list)
 #' @export
 test_dir <- function(path, filter = NULL, reporter = default_reporter(),
-                     env = test_env(), ..., encoding = "unknown", load_helpers = TRUE) {
+                     env = test_env(), ..., encoding = "unknown", 
+                     load_helpers = TRUE, wrap = TRUE) {
```

Comment:
No because it is inherited from `test_file`  (@inheritParams).  You can see it in the Rd file: https://github.com/hadley/testthat/pull/597/files#diff-f540ef3582dc78a36153378451f7b979.